app/src/main/java/practicum/getfitla_v3/NutritionItemFormat.java

```diff
@@ -1,46 +1,117 @@
 package practicum.getfitla_v3;
+import android.os.Parcel;
+import android.os.Parcelable;
 
-import android.support.v7.app.AppCompatActivity;
-
-//Class that formats the data into variables that are more easily called
-public class NutritionItemFormat {
+//Formats Information to be easily called
+public class NutritionItemFormat implements Parcelable {
     private int id;
-    private String title;
+    private String name;
     private String shortdesc;
-    private double rating;
-    private double price;
+    private String price;
```

Comment:
What's the reasoning for price being an String?

Reply:
For $

Reply:
For displaying, sure, but the data itself is a double. Only convert it to a string on display.

Reply:
That and if they accidentally input $ or another symbol then it won't cause an error

Reply:
@justinnhli they aren't going to need the number, other than to display it, so is it really necessary that they have it stored as a double?

Reply:
In general it's better to follow the type of the data. It's less surprising for someone reading it, and there's always the possibility that in the future it will be used in adding, etc.